### PR TITLE
Ignore ignore setting for orgs

### PR DESCRIPTION
**Pull request by prayerslayer**
Some orgs may have different needs than others even on GHE instances.

### CODE REVIEW

File: config/app-enterprise.yaml [hunk 1]
```diff
@@ -15,6 +15,14 @@ IGNORE_USER_CONFIG:
   - "approvals.pattern"
   - "approvals.veto"
   - "specification"
+# quick hack due to internal needs,
+# should be replaced with a more
+# general rule framework.
+IGNORE_USER_CONFIG_NOT_IN_ORGS:
```
Comment (ChristianLohmann) [inline config/app-enterprise.yaml]: are those the only ones who need 6 eyes?

**Comment by ChristianLohmann** [comment]
👍

File: config/app-enterprise.yaml (hunk 1 above)
Comment (prayerslayer) [inline config/app-enterprise.yaml]: apparently yes. if not we change it :)

**Comment by ChristianLohmann** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Pull request merged**